fdmi/source_dock.c

```diff
@@ -209,10 +209,21 @@ M0_INTERNAL void m0_fdmi__rec_id_gen(struct m0_fdmi_src_rec *src_rec)
 	M0_LEAVE();
 }
 
-M0_INTERNAL void m0_fdmi__record_post(struct m0_fdmi_src_rec *src_rec)
+M0_INTERNAL void m0_fdmi__enqueue(struct m0_fdmi_src_rec *src_rec)
```

Comment:
this function name includes 2 underscore. ca we make it 1 for naming consistency?

Reply:
Thank you. 
More explanation:
The function is declared in fdmi/source_dock_internal.h, and supposed to be used only by FDMI internally.
So, its name has a two-underscore, which means it is not exported to other module in Motr, and it is not exported to Motr client library.